base/db/R/insert.format.vars.R

```diff
@@ -71,47 +73,51 @@ insert.format.vars <- function(con, format_name, mimetype_id, notes = NULL, head
   }
 
   # Test if notes are a character string
-  if(!is.character(notes)&!is.null(notes)){
+  if(!is.character(notes) & !is.null(notes)){
     PEcAn.logger::logger.error(
       "Notes must be of type character"
     )
   }
 
   ######## Formats-Variables tests ###############
   if(!is.null(formats_variables)){
-    for(i in 1:nrow(formats_variables)){
-      if(!is.numeric(formats_variables[[i,"variable_id"]])){
+    for(i in seq_len(nrow(formats_variables))){
+      row_i <- formats_variables[i, ]
+      if(!is.numeric(row_i[["variable_id"]])){
         PEcAn.logger::logger.error(
           "variable_id must be an integer"
         )
       }
 
-      if(suppress == FALSE){
-        ## Test if variable_id already exists ##
-        var_id_test <- dplyr::tbl(con, "variables") %>% dplyr::select("id") %>% dplyr::filter(.data$id %in% !!formats_variables[[i, "variable_id"]]) %>% dplyr::collect(.data$id)
-        if(!is.null(var_id_test[1,1])){
+      if(identical(suppress, FALSE)){
```

Comment:
am I missing a reason this can't just be
```suggestion
      if (!suppress) {
```
?